src/lsm/manifest_log.zig

```diff
@@ -78,29 +77,44 @@ pub fn ManifestLogType(comptime Storage: type, comptime TableInfo: type) type {
             // However, we still store Label ahead of TableInfo to save space on the network.
             // This means we store fewer entries per manifest block, to gain less padding,
             // since we must store entry_count_max of whichever array is first in the layout.
-            // For a better understanding of this decision, see block_size() below.
+            // For a better understanding of this decision, see Block.size() below.
             assert(@sizeOf(TableInfo) % alignment == 0);
         }
 
-        const block_body_size = config.block_size - @sizeOf(vsr.Header);
-        const entry_size = @sizeOf(Label) + @sizeOf(TableInfo);
-        const entry_count_max_unaligned = @divFloor(block_body_size, entry_size);
-        pub const entry_count_max = @divFloor(entry_count_max_unaligned, alignment) * alignment;
+        /// The maximum number of table updates to the manifest by a half-measure of table
+        /// compaction (not including manifest log compaction).
+        ///
+        /// Input tables are removed from the manifest.
+        /// Output tables are removed from the manifest.
```

Comment:
Output tables are inserted into the manifest?

Reply:
Thanks, typo.